handlers/identity_test.go

```diff
@@ -16,7 +16,7 @@ import (
 
 	healthcheck "github.com/ONSdigital/dp-api-clients-go/health"
 	clientsidentity "github.com/ONSdigital/dp-api-clients-go/identity"
-	dphttp "github.com/ONSdigital/dp-net/http"
+	dphttp "github.com/ONSdigital/dp-net/v2/http"
 	dprequest "github.com/ONSdigital/dp-net/request"
```

Comment:
Maybe you can update this to `github.com/ONSdigital/dp-net/v2/request` , then we will have no more dependencies with v1, and you can run `go mod tidy` so that the dependency with v1 of dp-net in go.mod will disappear.

Reply:
Good shout, have updated this now ^^